worlds/ladx/__init__.py

```diff
@@ -114,7 +115,7 @@ class LinksAwakeningWorld(World):
 
     def convert_ap_options_to_ladxr_logic(self):
         self.player_options = {
-            option: getattr(self.multiworld, option)[self.player] for option in self.option_definitions
+            option: getattr(self.options, option) for option in dir(self.options) if not option.startswith("__") and option != "as_dict" # it's easier to also exclude the collection in the dict as using that [dir(self.options) is a list of elements]

```

Comment:
Did `[getattr(self.options, field.name) for field dataclasses.fields(self.options_dataclass)]` or something along those lines not give you what you wanted here?
Maybe I'm misunderstanding